v1/api/swagger/components/schemas/NewHost.yaml

```diff
@@ -149,3 +149,11 @@ properties:
     example:
       serviceType: BMaaS
       region: us-west-2
+  WWPNs:
+    type: array
+    description: FC host port world wide port names
+    items:
+      type: string
+    example:
+      - 18:31:01:02:ad:f2:a8:e5
+      - 19:31:01:02:ad:f2:a8:e5
```

Comment:
If they're mocked, then replace with real WWPNs as example.

Reply:
I have taken the real example from an email sent by Prashanth for existing WWPNs.  
20:31:01:02:ad:f2:a8:e5
21:31:01:02:ad:f2:a8:e5
I have just replaced 20 and 21 with 18 and 19 for capturing here as an example.

Reply:
The first octet contains vendor information that can not be always replaced with a mocked value.

Reply:
Done.